### ISSUE TITLE
Performance of trained Mobilenet_thin is not as good as the demo

### ISSUE THREAD

**Issue by Aileenlingyu** [ildoonet/tf-pose-estimation#131]
Hi, I trained mobilenet_thin version, the loss during training is as below:
last layer training loss: 43.82 
last layer validation loss： 46.06
total loss on training set: 268
total loss on validation set:286.2
But when I tested the model, there are a lot of errors, always missing one ankle joint, see attached result, it seems not to be as good as the demo version, I also plot the last layer validation loss with regard to each joint, and each part association, it seems to be similar with the demo model, cannot figure out the reason.

Could anyone please advise? Is my loss too large or something with the post-processing part? Many thanks!
![result1](https://user-images.githubusercontent.com/22608026/38041868-e00cc2d6-3280-11e8-8609-c46db7cbd6d0.png)
![result2](https://user-images.githubusercontent.com/22608026/38041871-e1b7fb0a-3280-11e8-97fa-c6390f65eae0.png)
![barplot1](https://user-images.githubusercontent.com/22608026/38041875-e480f530-3280-11e8-8613-de3e44bde837.png)
![barplot2](https://user-images.githubusercontent.com/22608026/38041877-e563ef66-3280-11e8-8514-d79680949cb5.png)

**Comment by ouceduxzk** [comment]
I would say that the total loss is a little big, should be < 230, here is my mobilenet_thin plot on ski.jpg, maybe you should try with smaller batch size, what exactly the training command you are using?
[
![mobilenet_thinski](https://user-images.githubusercontent.com/2516683/38043647-fea48dc8-32b7-11e8-93b2-83087a32c503.jpg)
](url)

**Comment by Aileenlingyu** [comment]
Thanks for the information! 
I am using
batch size = 16
Learning rate = 0.0001
Optimizer: RMSprop
Input width: 368
Input height: 368

**Comment by ouceduxzk** [comment]
i am using batch size 48, find it quite good, did not tried much with RMSprop, I find adam better though

**Comment by Aileenlingyu** [comment]
Thank you so much! I will try it!

**Comment by ouceduxzk** [comment]
some tips, the loss sometimes is not very indicative after it below 250, so you should constantly check the heatmap plot , sometimes i found the heatmap plot is chaotic and no matter how long it trained, the nice visualizaiton does not come back even though the loss looks good.

**Comment by seovchinnikov** [comment]
were you able to resolve the problem? I cant reproduce these results too with the same config as @ouceduxzk

**Comment by Aileenlingyu** [comment]
Me neither, still trying

**Comment by ouceduxzk** [comment]
the demo result visually looks reproducible to me

**Comment by Aileenlingyu** [comment]
@ouceduxzk Hi, may I ask how many iterations did you run? I am currently running 60k iterations, but not found any significant decrease since 40k iterations. Thanks much!

**Comment by ouceduxzk** [comment]
around 100k, one thing i forget to mention, for the kernel size of branches, i adopted the cmupose way of using 7 x7 instead of 3x3

**Comment by Aileenlingyu** [comment]
Thank you! I am also trying to increase the complexity of the model, hope it can work out.
